### PR TITLE
Match against non-lower case strings

### PR DESCRIPTION
**Pull request by jesstelford**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = function falsey(val, keywords) {
     ? ['none', 'nil', 'nope', 'no', 'nada', '0', 'false']
     : arrayify(keywords);
 
-  return arr.indexOf(val) !== -1;
+  return arr.indexOf(val.toLoweCase ? val.toLowerCase() : val) !== -1;
 };
```
Comment (jonschlinkert) [inline index.js]: there is a typo in `toLoweCase`.

**Comment by jonschlinkert** [comment]
Hi, thanks for the pr. Looks like some of the tests are failing, I haven't had a chance to investigate why. Were the tests passing locally?

**Comment by jesstelford** [comment]
Whoops - that typo caused the tests to fail!

**Comment by alexframe** [comment]
@jonschlinkert was there any update on getting this merged and published?

**Pull request merged**